site/README.md

```diff
@@ -1,18 +0,0 @@
-# IPFS Distributions
-
-
-## Development
-
-Harpjs setup with
-
-* Live reload using browser-sync
-* JavaScript
-  * bundeling using webpack
-  * transpiling using babel
-* Linting using eslint
-
-
-```bash
-$ gulp
-$ open localhost:3000
-```
```

Comment:
meaning that a user MUST be able to come into this repo, update a version, enter `make` or something close to that, and have everything work. if a user needs to even think about npm, the constraint is not met. `make` should install everything it needs.